server_admin/topics/clients/proc-creating-oidc-client.adoc

```diff
@@ -0,0 +1,116 @@
+tab.
+
+.Client Settings
+image:{project_images}/client-settings-oidc.png[Client Settings]
+
+Let's walk through each configuration item on this page.
+
+*Client ID*
+
+The alpha-numeric string that will be used as the client identifier for OIDC requests.
+
+*Name*
+
+The display name for the client whenever it is displayed in a {project_name} UI screen.  You can localize
+the value of this field by setting up a replacement string value i.e. $\{myapp}.  See the link:{developerguide_link}[{developerguide_name}]
+for more information.
+
+*Description*
+
+The description of the client.  This can also be localized.
+
+*Enabled*
+
+When turned off, the client will not be allowed to request authentication.
+
+*Consent Required*
+
+If this is on, then users will get a consent page which asks the user if they grant access to that application.  It will also display the metadata that the client is interested in so that the user knows exactly what information the client is getting access to.
+If you have ever done a social login to Google, you'll often see a similar page.  {project_name} provides the same functionality.
+
+[[_access-type]]
+*Access Type*
+
+The type of OIDC client.
+
+_confidential_::
+  Confidential access type is for server-side clients that need to perform a browser login and require a client secret when they turn an access code into an access token,
+  (see https://tools.ietf.org/html/rfc6749#section-4.1.3[Access Token Request] in the OAuth 2.0 spec for more details). This type should be used for server-side applications.
+
+_public_::
+  Public access type is for client-side clients that need to perform a browser login. With a client-side application there is no way to keep a secret safe. Instead it is very important to restrict  access by configuring correct redirect URIs for the client.
+
+_bearer-only_::
+  Bearer-only access type means that the application only allows bearer token requests.
+  If this is turned on, this application cannot participate in browser logins.
+
+*Standard Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Authorization Code Flow>>.
+
+*Implicit Flow Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Implicit Flow>>.
+
+*Direct Access Grants Enabled*
+
+When enabled, clients are allowed to use the OIDC <<_oidc-auth-flows,Direct Access Grants>>.
+
+*Root URL*
+
+If {project_name} uses any configured relative URLs, this value is prepended to them.
+
+*Valid Redirect URIs*
+
+This is a required field.  Enter in a URL pattern and click the + sign to add.  Click the - sign next to URLs you want to remove.
+Remember that you still have to click the `Save` button!
+Wildcards (*) are only allowed at the end of a URI, i.e. $$http://host.com/*$$
+
+You should take extra precautions when registering valid redirect URI patterns. If you make
```

Comment:
What precautions? 
I would rephrase this something to avoid doing or something that increases security. 

"Exclusive redirect URI patterns are typically more secure. See..."